src/platform/user/profile/vap-svc/components/base/VAPServiceEditModalErrorMessage.jsx

```diff
@@ -63,30 +62,44 @@ export default function VAPServiceEditModalErrorMessage({
 
     default:
       content = (
-        <p>
+        <p id="error-message-details">
           We’re sorry. We can’t update your information right now. We’re working
           to fix this problem. Please check back later.
         </p>
       );
   }
 
   return (
-    <AlertBox
-      backgroundOnly
-      status="error"
-      className="va-profile-alert vads-u-margin-y--1"
-    >
-      <div className="vads-u-display--flex">
-        <i
-          className="fas fa-info-circle vads-u-font-size--md vads-u-color--black vads-u-margin-right--2 vads-u-padding-top--0p5"
-          aria-hidden="true"
-          role="img"
-        />
-        <span className="sr-only">Alert: </span>
-        <div role="alert" aria-live="polite">
-          {content}
+    <>
+      <va-alert
+        background-only
+        close-btn-aria-label="Close notification"
+        status="error"
+        visible
+      >
+        <div className="vads-u-display--flex vads-u-align-items--baseline vads-u-margin-left--neg1p5 vads-u-margin-y--neg1p5">
+          <i
+            className="fas fa-info-circle vads-u-font-size--md vads-u-color--black vads-u-margin-right--2 vads-u-padding-top--0p5"
+            aria-hidden="true"
+            role="img"
+          />
+          <span className="sr-only">Alert: </span>
+          <div role="alert" aria-live="polite">
```

Comment:
The `va-alert` wraps everything in a div with `aria-live="assertive"`. Will it cause problems for screen readers if this also has an `aria-live="polite"`?

Reply:
![Screen Shot 2022-06-29 at 9 33 51 AM](https://user-images.githubusercontent.com/1793923/176449382-90cbd5fb-c3e7-4df1-bfbd-27a6b6a32751.png)

I don't see where you see an `aria-live="assertive:"` do you have an example?

Reply:
![Screen Shot 2022-06-29 at 9 00 47 AM](https://user-images.githubusercontent.com/13838621/176455793-20bba934-4309-4f1e-8bac-cb0f744c32d4.png)

Reply:
It's on the `<div role="alert"` element. `aria-live="assertive"` is only applied to `va-alert` when `status="error"` , which is why it's not present in your example

Reply:
Currently the alert content gets read twice for me when using VoiceOver on macOS

Reply:
I removed the `polite` for now. Thanks for the catch!